docs/system/k8s-metrics.md

```diff
@@ -162,6 +200,44 @@ This metric is [recommended][MetricRecommended].
 <!-- markdownlint-disable -->
 
 
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+### Metric: `k8s.node.memory.usage`
+
+This metric is [recommended][MetricRecommended].
+
+<!-- semconv metric.k8s.node.memory.usage(metric_table) -->
```

Comment:
Additionally - the `(metric_table)` portion of snippets is optional / no longer needed.